src/App.jsx

```diff
@@ -1,8 +1,96 @@
 import './App.css'
+import ButtonItem from "./components/buttons/ButtonItem.jsx";
+import ProductItemTile from "./components/productItemTile/ProductItemTile.jsx";
+import ArticleStoryTile from "./components/articleStory/ArticleStoryTile.jsx";
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
```

Comment:
Blijf letten op de HTML structuur. Je hebt keurig een main en een footer element, maar waar is de header element?